convert2rhel/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.16"
```

Comment:
Not big deal, but we need to figure out how to set this version automatically, other wise we could miss to change this file.

Reply:
it is not differ from the previous way, where we needed to specify the version in two places: setup.py and rpm spec. Now instead of setup.py it is convert2rhel/__init__.py.
However, not ideal. There is https://github.com/fedora-python/pyp2rpm that I wish to try one day

Reply:
I'm fine with moving the version out of setup.py. This solution seems to me actually even cleaner.

Reply:
Just add a new line here please.